WordPress/Sniffs/Theme/AddThemeSupportTagsCheck1Sniff.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
+/**
+ * WordPress_Sniffs_Theme_AddThemeSupportTagsCheckSniff.
+ *
+ * Verify that an add_theme_support() call is made for any feature the
+ * theme has been tagged with from the following list: custom-background,
+ * custom-header, custom-menu, featured-images/post-thumbnails, post-formats,
+ * custom-logo.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    khacoder
+ */
+class WordPress_Sniffs_Theme_AddThemeSupportTagsCheck1Sniff extends WordPress_AbstractThemeSniff {
```

Comment:
We can remove the word "Check" from the class name. Sniff is sufficient. We should remove the number. We can change the name of the other class.